cpp/create_full_text_db.cc

```diff
@@ -137,28 +128,29 @@ void ProcessRecords(const unsigned max_record_count, const unsigned skip_count,
 
     std::cout << "Skip " << skip_count << " records\n";
 
-    while (MarcUtil::Record record = MarcUtil::Record::XmlFactory(input)) {
-        record.setRecordWillBeWrittenAsXml(true);
+    long record_start = input->tell();
```

Comment:
Should be off_t now.  File:tell() returned a "long" which only portably supports files with up to 2GiB in size.